frontend/src/pages/_document.tsx

```diff
@@ -4,7 +4,7 @@ export default function Document() {
   return (
     <Html lang="en"> 
       <Head> 
-        <link rel="icon" href="pong-nyan.png"/> 
+        <link rel="icon" href="pongNyan.png"/> 
```

Comment:
```suggestion
        <link rel="icon" href="pong-nyan.png"/> 
```
url 관련된 곳에 camel case 나오면 불안해요